suite-native/atoms/src/Sheet/BottomSheetFlashList.tsx

```diff
@@ -0,0 +1,146 @@
+    paddingHorizontal: utils.spacings.sp16,
+}));
+
+const sheetWithOverlayStyle = prepareNativeStyle(_ => ({
+    flex: 1,
+    justifyContent: 'flex-end',
+}));
+
+export const BottomSheetFlashList = <TItem,>({
+    isVisible,
+    isCloseDisplayed = true,
+    onClose,
+    title,
+    subtitle,
+    ExtraProvider,
+    estimatedListHeight = 0,
+    ...flashListProps
+}: BottomSheetFlashListProps<TItem>) => {
+    const { applyStyle } = useNativeStyles();
+    const insets = useSafeAreaInsets();
+    const {
+        animatedSheetWithOverlayStyle,
+        animatedSheetWrapperStyle,
+        closeSheetAnimated,
+        openSheetAnimated,
+        panGestureEvent,
+        scrollEvent,
+    } = useBottomSheetAnimation({
+        onClose,
+        isVisible,
+    });
+    const panGestureRef = useRef(null);
+    const scrollViewRef = useRef(null);
+
+    useEffect(() => {
+        if (isVisible) {
+            openSheetAnimated();
+        }
+    }, [isVisible, openSheetAnimated]);
+
+    const handleCloseSheet = () => {
+        closeSheetAnimated();
+    };
+
+    const handlePressOutside = (event: GestureResponderEvent) => {
+        if (event.target === event.currentTarget) closeSheetAnimated();
+    };
+
+    const insetBottom = Math.max(insets.bottom, DEFAULT_INSET_BOTTOM);
+
+    return (
+        <BottomSheetContainer
+            isVisible={isVisible}
+            onClose={handleCloseSheet}
+            ExtraProvider={ExtraProvider}
+        >
+            <Animated.View
+                style={[animatedSheetWithOverlayStyle, applyStyle(sheetWithOverlayStyle)]}
+            >
+                <Pressable style={applyStyle(sheetWithOverlayStyle)} onPress={handlePressOutside}>
+                    <PanGestureHandler
+                        ref={panGestureRef}
+                        activeOffsetY={5}
+                        failOffsetY={-5}
+                        onGestureEvent={panGestureEvent}
+                    >
+                        <Animated.View
+                            style={[
+                                animatedSheetWrapperStyle,
+                                applyStyle(sheetWrapperStyle, {
+                                    insetBottom,
+                                }),
+                            ]}
+                        >
+                            <BottomSheetHeader
+                                title={title}
+                                subtitle={subtitle}
+                                isCloseDisplayed={isCloseDisplayed}
+                                onCloseSheet={closeSheetAnimated}
+                            />
+
+                            <Box
+                                style={{
+                                    // only estimated height is used, so we need to add some buffer to prevent unnecessary scrolling, when list is shorter than estimated
+                                    height: estimatedListHeight * 1.1,
```

Comment:
I'd prefer the multiplier to be first. Not a big deal.
```suggestion
                                    height: 1.1 * estimatedListHeight,
```